core/src/main/java/com/ritense/valtimo/service/CamundaTaskService.java

```diff
@@ -269,6 +269,7 @@ public void complete(String taskId) {
         Hibernate.initialize(task.getVariableInstances());
         Hibernate.initialize(task.getIdentityLinks());
         entityManager.detach(task);
+        task.getIdentityLinks().forEach(entityManager::detach);
```

Comment:
Valtimo sometimes tries to save the identitylinks which can result in errors

Reply:
Can you add that as a comment to the code?

Reply:
Added